artifactory/cli.go

```diff
@@ -2872,10 +3033,27 @@ func overrideIntIfSet(field *int, c *cli.Context, fieldName string) {
 	}
 }
 
-func validateBuildParams(buildConfig *utils.BuildConfiguration) error {
-	if (buildConfig.BuildName == "" && buildConfig.BuildNumber != "") || (buildConfig.BuildName != "" && buildConfig.BuildNumber == "") || (buildConfig.Module != "" && buildConfig.BuildName == "" && buildConfig.BuildNumber == "") {
-		return errors.New("The build-name, build-number and module options cannot be sent separately.")
+func validateCommonContext(c *cli.Context) error {
```

Comment:
I couldn't see any usage for that function

Reply:
Playground tests, removed.